.github/workflows/publish.yml

```diff
@@ -46,27 +56,36 @@ jobs:
         # Usually, the artifact that cannot be uploaded is strangely already
         # in the repo. As the result, by ignoring the exception, we should end
         # up with a working release in most cases.
-        run: >-
-          ./gradlew --build-cache publish
-          --continue
-          --info
-          -PciBuild=true
-          -Partifacts.itemis.cloud.user=${{ secrets.ARTIFACTS_ITEMIS_CLOUD_USER }}
-          -Partifacts.itemis.cloud.pw=${{ secrets.ARTIFACTS_ITEMIS_CLOUD_PW }}
-          -Pgpr.user=${{ github.actor }}
-          -Pgpr.key=${{ secrets.GITHUB_TOKEN }}
-          -Pgpr.universalkey=${{ secrets.GHP_UNIVERSAL_PUBLISH_TOKEN }}
-          -Porg.gradle.internal.http.connectionTimeout=180000
-          -Porg.gradle.internal.http.socketTimeout=180000
+        run: |-
+          if [[ "$IS_PR" = 'true' ]]; then
+            TARGET=publishToMavenLocal
+          else
+            TARGET=publish
```

Comment:
If we publish locally on each PR we do not need the "build.yaml" workflow, rigth?
Should we merge `build.yaml` and `publish.yaml`?

Reply:
That makes the trigger conditions a bit more complicated as the build also runs on main.

Reply:
With these changes `publish.yaml`won't trigger on each PR. Only if the file itself was changed.

Reply:
So what shall we do about this? I'd say dropping this workflow and putting it into the build workflow has a higher potential for accidentally publishing undesired stuff to the Maven repos when fiddling with the build workflow.

Reply:
> That makes the trigger conditions a bit more complicated as the build also runs on main.

> With these changes publish.yamlwon't trigger on each PR. Only if the file itself was changed.

> I'd say dropping this workflow and putting it into the build workflow has a higher potential for accidentally publishing undesired stuff to the Maven repos when fiddling with the build workflow.

I agree with your points. Let's keep `build.yaml` and `publish.yaml` separate.